src/components/Button/__tests__/index-test.js

```diff
@@ -1,5 +1,52 @@
 /* eslint-env jasmine, jest */
 
+import React from 'react';
+import Button from '..';
+import { mount, shallow } from 'enzyme';
+
+const findNativeButton = wrapper => wrapper.find('[role="button"]');
```

Comment:
Can you use something like `getDOMNode` instead of leaking knowledge about how `accessibilityRole` is implemented into this test?